CONTRIBUTING.md

```diff
@@ -1,2 +1,2 @@
-To add headers for a new release run "rake add_source" with VERSION and PATCHLEVEL provide.
+To add headers for a new release, run "rake add_source" with the VERSION and PATCHLEVEL provided. Note that the PATCHLEVEL variable is optional.
```

Comment:
Does this change also apply to the upstream version? Can it be upstreamed via a PR to them?